sycl/include/sycl/ext/intel/esimd/simd.hpp

```diff
@@ -125,9 +125,30 @@ class simd : public detail::simd_obj_impl<
     return sycl::ext::oneapi::experimental::simd<Ty, N1>(base_type::data());
   }
 
+  /// @ingroup sycl_esimd_misc
+  /// Prints a \c simd object to an output stream.
```

Comment:
this is not what the code is doing now, it rather converts the object to std::string